src/features/email/lib/project-invitation.ts

```diff
@@ -0,0 +1,71 @@
+import { createTransport } from "nodemailer";
+import { Resend } from "resend";
+
+import { projectInvitationTemplate } from "@/src/features/email/templates/ProjectInvitation";
+
+export const sendProjectInvitation = async (
+  to: string,
+  senderName: string,
+  projectName: string,
+) => {
+  try {
+    const transportChannel = process.env.EMAIL_TRANSPORT_CHANNEL ?? "smtp";
+    const html_template = projectInvitationTemplate(
+      senderName,
+      to,
+      projectName,
+    );
+
+    const fromName = process.env.EMAIL_FROM_NAME ?? "Langfuse";
+    const fromAddress =
+      process.env.EMAIL_FROM_ADDRESS ?? "team_langfuse@langfuse.com";
```

Comment:
Use env. instead of process env for zod-typed environments
`import { env } from "@/src/env.mjs"`